Assets/Scripts/Controllers/PlayController.cs

```diff
@@ -129,13 +126,15 @@ public void EndLevel()
         bool pizzaCorrect = currPizza.IsOrderCorrect(currPizzaOrder);
         if (pizzaCorrect)
         {
-            score += 4;
-            if (score > MAX_SCORE)
+            // TODO: think about how we want to handle communication across components?
+            // maybe we want to maek a score manager and an event system that triggers score controller?
+            // we could lessen our dependency on singletons that way maybe
+            if (ScoreController.scInstance.UpdateScore(1) > Constants.MAX_SCORE)
```

Comment:
Should this check of > max score occur in play controller?

Reply:
Yeah this would probably be better to be in the `UpdateScore` function, but I don't think it's a huge deal